Platform/CoffeelakeBoardPkg/CfgData/CfgData_Memory.dsc

```diff
@@ -68,7 +76,7 @@
   # !BSF NAME:{Dqs Pins Interleaved Setting} TYPE:{Combo} OPTION:{$EN_DIS}
   # !BSF HELP:{Indicates DqPinsInterleaved setting: board-dependent}
   gCfgData.DqPinsInterleaved                |      * | 0x01 | 0x1
-  
+
   # !BSF NAME:{Tseg Size} TYPE:{Combo}
```

Comment:
This Tseg has invalid value(0x800000) and I think this should be removed and controlled in code directly because
there is description "0x400000 for Release build and 0x1000000 for Debug build" in FspmUpd.h.
Let me do it in separate commit.

Reply:
Thanks!  Sure, let us fix that separately.